tock/tock/static/js/chosen/bower.json

```diff
@@ -1,20 +1,6 @@
 {
```

Comment:
It'd be ideal if these libs came from npm, but for now, it might help a bit if they were in a `vendor` directory, so at least all contributors know they are vendor libs and not src code.

Reply:
Totally agree, but that's going to be a longer-term solution and ties into the other change suggested as well (combining them all into one file, probably with something like Webpack and just making it a part of the build/deployment process).

Reply:
Yea that sounds fine